modules/http_push.c

```diff
@@ -49,6 +50,7 @@ static int nw_http_push_start_acquire_data(struct nodewatcher_module *module,
 
   /* Get the configured URL from UCI. */
   char *url = nw_uci_get_string(uci, "nodewatcher.@agent[0].push_url");
+
```

Comment:
Readability habit I guess... You want me to stop doing that or just wanted an explanation? :)